app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="row justify-content-center">
+  <div class="col-10 mt-5">
+    <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'form-control m-2', placeholder: 'username' %>
```

Comment:
- [ ] Kindly change the placeholder here to `email`, so that users know the log in expects an email, not username

Reply:
hey @DammyShittu, I wrote username because it was asked in the [wireframe ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_login.png). But I can change that 
😊